packages/web3-utils/types/tests/sha3-test.ts

```diff
@@ -15,30 +15,31 @@
     along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 /**
- * @file sha3-tests.ts
+ * @file keccak256-tests.ts
```

Comment:
If sha3 is staying as a alias you should be keep these tests and creating a new file test for keccak256 tests,  both still need to be tested.

Reply:
okay cool.